main.go

```diff
@@ -1,17 +1,91 @@
 package main
 
 import (
+	"fmt"
```

Comment:
Lots of top-level dependencies added here. How will this interact with testing? Would it be better to set up the init function to follow the dependency injection patterns?

Reply:
As far as I know, the `init()` function has to be in the main package, and in the same file as the `main()` function.  I believe that this happens because the `init()` function executes *before* the `main()`.

I've added some comments next to each import explaining why I needed to import them.

Regarding the test, this imports doesn't affect the testing at all